zendesk/1.0.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,74 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 1.0.0
+name: Zendesk Integration
+displayName: Zendesk Integration
+createdAt: 2022-02-04T00:00:00Z
+description: Keptn integration with Zendesk.com
+logoURL: https://en.wikipedia.org/wiki/Zendesk#/media/File:Zendesk_logo.svg
```

Comment:
I think https://upload.wikimedia.org/wikipedia/commons/thumb/c/c8/Zendesk_logo.svg/1280px-Zendesk_logo.svg.png should be used here.